server/controllers/notification.js

```diff
@@ -0,0 +1,82 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /notifications/
+// @desc create a new notification
+// @access Public
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const { type, title, description } = req.body;
```

Comment:
```suggestion
  const { type, title, description, receiver } = req.body;
```
Based on the updates to the model